typescript/schemas/src/internal/schemas.ts

```diff
@@ -1,5 +1,57 @@
 import { FoxgloveEnumSchema, FoxgloveMessageSchema } from "./types";
 
+const AudioFormat: FoxgloveEnumSchema = {
+  type: "enum",
+  name: "AudioFormat",
+  parentSchemaName: "Audio",
+  protobufEnumName: "AudioFormat",
+  description:
+    "An enumeration of supported audio formats",
+  values: [
+    { value: 0, name: "PCM_S16", description: "Interleaved, signed 16-bit PCM" },
+    { value: 1, name: "OPUS", description: "Opus" },
+    { value: 2, name: "MP4A_40_2", description: "MPEG-4 AAC LC" },
+  ],
+};
+
+const Audio: FoxgloveMessageSchema = {
+  type: "message",
+  name: "Audio",
+  description: "A single frame of an audio bitstream",
+  fields: [
+    {
+      name: "timestamp",
+      type: { type: "primitive", name: "time" },
+      description: "Timestamp of audio frame",
+    },
+    {
+      name: "data",
+      type: { type: "primitive", name: "bytes" },
+      description: `Audio frame data`,
```

Comment:
Is there more we need to say about the data here? For context look at the details we provide on `CompressedVideo` formats and the things we expect. Is that true here as well? Anything about the amount or lack thereof that should or should not be present here? Anything detailed per-format that we should indicate?